api/swagger.yaml

```diff
@@ -4187,11 +4187,11 @@ paths:
         - name: "path"
           in: "query"
           required: true
-          description: "Path to a directory in the container to extract the archive’s contents into. "
+          description: "Path to a directory in the container to extract the archive's contents into. "
           type: "string"
         - name: "noOverwriteDirNonDir"
           in: "query"
-          description: "If “1”, “true”, or “True” then it will be an error if unpacking the given content would cause an existing directory to be replaced with a non-directory and vice versa."
+          description: "If `1`, `true`, or `True` then it will be an error if unpacking the given content would cause an existing directory to be replaced with a non-directory and vice versa."
```

Comment:
Have you tried if this renders correctly if you `make swagger-docs`, and view the generated documentation?